I cant start the program

Hello i cant start the program can you help me
i have this error

 epicgames-freebies-claimer@1.4.0 start /home/pi/epicgames-freebies-claimer
> node claimer.js "antoinerzad2@gmail.com" "password" "0"

internal/modules/cjs/loader.js:1023
  throw err;
  ^

Error: Cannot find module 'check-update-github'
Require stack:
- /home/pi/epicgames-freebies-claimer/claimer.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1020:15)
    at Function.Module._load (internal/modules/cjs/loader.js:890:27)
    at Module.require (internal/modules/cjs/loader.js:1080:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/home/pi/epicgames-freebies-claimer/claimer.js:3:21)
    at Module._compile (internal/modules/cjs/loader.js:1176:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1196:10)
    at Module.load (internal/modules/cjs/loader.js:1040:32)
    at Function.Module._load (internal/modules/cjs/loader.js:929:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js                                                                                                                     :71:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/pi/epicgames-freebies-claimer/claimer.js' ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! epicgames-freebies-claimer@1.4.0 start: `node claimer.js "antoinerzad2@                                                                                                                     gmail.com" "password" "0"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the epicgames-freebies-claimer@1.4.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional log                                                                                                                     ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2020-05-15T13_37_41_731Z-debug.log

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
         "eslint": "^6.8.0"
     },
     "dependencies": {
+        "check-update-github": "^0.0.4",
         "colors": "^1.4.0",
         "epicgames-client": "^2.0.23",
         "epicgames-client-login-adapter": "^0.1.0",

